### PR TITLE
docs: update pnpm commands, fix typos and grammar (#49990

### PR DESCRIPTION
**Pull request by Ksound22**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

The only significant change here is the directory to run `pnpm run create-project`. Every other thing is a typo or gramo nitpick

### CODE REVIEW

**Comment by ojeytonwilliams** [comment]
Hey, @Ksound22, thanks for all these improvements. Could you drop the first three commits, though?

**Comment by Ksound22** [comment]
> Hey, @Ksound22, thanks for all these improvements. Could you drop the first three commits, though?

on it

**Comment by Ksound22** [comment]
> Hey, @Ksound22, thanks for all these improvements. Could you drop the first three commits, though?

I couldn't find on of them, but I dropped the other two. MAybe that was what resulted into some merge conflicts again.

**Comment by ojeytonwilliams** [comment]
So, there are 4 commits left of which 3 are new. I'd do the following (starting from the PR branch)

```
git checkout main
git pull upstream main
git checkout -
git rebase -i main
```
You can drop the unwanted one during the rebase.

**Comment by Ksound22** [comment]
> So, there are 4 commits left of which 3 are new. I'd do the following (starting from the PR branch)
> 
> ```
> git checkout main
> git pull upstream main
> git checkout -
> git rebase -i main
> ```
> 
> You can drop the unwanted one during the rebase.

Everything should be good now. THanks.

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
Cheers, @Ksound22 the commits look good now.

File: docs/how-to-setup-freecodecamp-locally.md [hunk 1]
```diff
@@ -118,7 +118,84 @@ This is essential, as it allows you to work on your own copy of freeCodeCamp on
...
+   We highly recommend using [Visual Studio Code](https://code.visualstudio.com/) or [Atom](https://atom.io/). These are great, free, and open source code editors.
+
+4. Set up linting for your code editor.
+
+   You should have [ESLint running in your editor](http://eslint.org/docs/user-guide/integrations.html), and it will highlight anything that doesn't conform to [freeCodeCamp's JavaScript Style Guide](http://forum.freecodecamp.org/t/free-code-camp-javascript-style-guide/19121).
+
+   > [!TIP]
+   > Please do not ignore any linting errors. They are meant to **help** you and to ensure a clean and simple codebase.
+
+## Clone your Fork from GitHub
```
[the remark this answers is not in the document]
Comment (Ksound22) [reply docs/how-to-setup-freecodecamp-locally.md:198]: Screenshot 2023-04-07 at 15 52 21

Something's really wrong with it. What do I do?

[the remark this answers is not in the document]
Comment (ojeytonwilliams) [reply docs/how-to-setup-freecodecamp-locally.md:198]: Could you give me a little more context? What seems to be the problem?

[the remark this answers is not in the document]
Comment (Ksound22) [reply docs/how-to-setup-freecodecamp-locally.md:198]: Even after saving the file, there's an indication that there are changes not saved yet.

[the remark this answers is not in the document]
Comment (Ksound22) [reply docs/how-to-setup-freecodecamp-locally.md:198]: @ojeytonwilliams I'm not sure about what's happening. I think the problem came from a conflict. The lines from 121 to 198 seem not to be a part of the file at all. I can't just remove it. 

Do you have a way to check if those lines are not part of the file at all?

[the remark this answers is not in the document]
Comment (ojeytonwilliams) [reply docs/how-to-setup-freecodecamp-locally.md]: I've no idea what was going wrong, but I was able to delete it and commit the changes. It all looks good now!

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

File: docs/language-lead-handbook.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ This handbook will help you set up and use the tools for your localization effor
 
 ## How to invite new contributors to Ghost
```
Comment (ShaunSHamilton) [inline docs/language-lead-handbook.md:5]:
```suggestion
## How to Invite New Contributors to Ghost
```

File: docs/language-lead-handbook.md [hunk 3]
```diff
@@ -26,7 +26,7 @@ You can find the up-to-date build schedules and status here: https://github.com/
 
 ## How to mention the original author of a translated article
```
Comment (ShaunSHamilton) [inline docs/language-lead-handbook.md:27]:
```suggestion
## How to Mention the Original Author of a Translated Article
```

File: docs/language-lead-handbook.md [hunk 4]
```diff
@@ -39,22 +39,22 @@ With `link` being the link of the original article.
 ## How to update trending articles
```
Comment (ShaunSHamilton) [inline docs/language-lead-handbook.md:39]:
```suggestion
## How to Update Trending Articles
```

File: docs/language-lead-handbook.md [hunk 5]
```diff
@@ -39,22 +39,22 @@ With `link` being the link of the original article.
...
 
 There are two places in which to change the trending articles.
 
 - [The curriculum repository](https://github.com/freeCodeCamp/freeCodeCamp/)
 - [The CDN repository](https://github.com/freeCodeCamp/cdn)
 
-For each article you will need to create a shorter title to use in the footer.
+For each article, you will need to create a shorter title to use in the footer.
 
 ### Change trending articles in the curriculum
```
Comment (ShaunSHamilton) [inline docs/language-lead-handbook.md:51]:
```suggestion
### Change Trending Articles in the Curriculum
```

File: docs/language-lead-handbook.md [hunk 6]
```diff
@@ -80,9 +80,9 @@ You will want to [build the translated client locally](how-to-enable-new-languag
 
 ### How to update the trending articles in the cdn
```
Comment (ShaunSHamilton) [inline docs/language-lead-handbook.md:81]:
```suggestion
### How to Update the Trending Articles in the CDN
```

File: docs/language-lead-handbook.md [hunk 7]
```diff
@@ -106,7 +106,7 @@ You can convert from one format to the other carefully changing it manually. Or
 
 ## How to translate articles in the footer links
```
Comment (ShaunSHamilton) [inline docs/language-lead-handbook.md:107]:
```suggestion
## How to Translate Articles in the Footer Links
```

File: docs/language-lead-handbook.md [hunk 8]
```diff
@@ -306,7 +306,7 @@ The `motivationalQuotes` array is an array of objects, these objects should incl
 ```
 
 > [!TIP]
-> You should start with at least a dozen quotes, to have some variety. A new quote is shown every time the user reload the page.
+> You should start with at least a dozen quotes, to have some variety. A new quote is shown every time the user reloads the page.
 
 ## How to update the common links
```
Comment (ShaunSHamilton) [inline docs/language-lead-handbook.md:311]:
```suggestion
## How to Update the Common Links
```

File: docs/language-lead-handbook.md [hunk 9]
```diff
@@ -353,34 +353,34 @@ When you have finished setting this, press the Pre-Translate button and wait. It
 ## How to update Crowdin Glossary
```
Comment (ShaunSHamilton) [inline docs/language-lead-handbook.md:353]:
```suggestion
## How to Update Crowdin Glossary
```

File: docs/moderator-handbook.md [hunk 10]
```diff
@@ -94,7 +94,7 @@ Pull Requests (PRs) are how contributors submit changes to freeCodeCamp's reposi
 
    Some PRs are automated dependency updates made via an integration. You should not merge or approve these PRs. One of the dev-team members will take care of reviewing and merging such automated PRs.
 
-#### How to review, merge or close pull requests
+#### How to review, merge, or close pull requests
 
 ##### Assign yourself to a pull request:
```
Comment (ShaunSHamilton) [inline docs/moderator-handbook.md:99]:
```suggestion
##### Assign Yourself to a Pull Request:
```

File: docs/moderator-handbook.md [hunk 11]
```diff
@@ -94,7 +94,7 @@ Pull Requests (PRs) are how contributors submit changes to freeCodeCamp's reposi
 
    Some PRs are automated dependency updates made via an integration. You should not merge or approve these PRs. One of the dev-team members will take care of reviewing and merging such automated PRs.
 
-#### How to review, merge or close pull requests
+#### How to review, merge, or close pull requests
```
Comment (ShaunSHamilton) [inline docs/moderator-handbook.md:97]:
```suggestion
#### How to Review, Merge, or Close Pull Requests
```

File: docs/how-to-work-on-tutorials-that-use-coderoad.md [hunk 12]
```diff
@@ -2,7 +2,7 @@ This page describes how to contribute to the freeCodeCamp tutorials and projects
 
 ## How the tutorials work
```
Comment (ShaunSHamilton) [inline docs/how-to-work-on-tutorials-that-use-coderoad.md:3]:
```suggestion
## How the Tutorials Work
```

File: docs/how-to-work-on-the-docs-theme.md [hunk 13]
```diff
@@ -44,20 +44,14 @@ Typically you would not need to change any configuration or build the site local
...
 - The sidebar navigation is generated from [`_sidebar.md`](_sidebar.md).
 
-### Serving the documentation site locally
+### Serving the Documentation Site Locally
 
-Install freeCodeCamp locally ([see the local setup guide](how-to-setup-freecodecamp-locally)), we bundled the CLI with the development tools so you can run any of the below commands as needed from the root of the repo:
+Install freeCodeCamp locally ([see the local setup guide](how-to-setup-freecodecamp-locally)), we bundled the CLI with the development tools so you can run the command below as needed from the root of the repo:
 
-#### Serve and launch the documentation site only
+#### Serve and launch the documentation site
```
Comment (ShaunSHamilton) [inline docs/how-to-work-on-the-docs-theme.md:51]:
```suggestion
#### Serve and Launch the Documentation Site
```

File: docs/how-to-work-on-coding-challenges.md [hunk 14]
```diff
@@ -397,7 +397,7 @@ Take the following steps when adding a new challenge hints/solutions related top
 
 ### Guidelines for content of hints and solutions topics
```
Comment (ShaunSHamilton) [inline docs/how-to-work-on-coding-challenges.md:398]:
```suggestion
### Guidelines for Content of Hints and Solutions Topics
```

File: docs/how-to-setup-freecodecamp-mobile-app-locally.md [hunk 15]
```diff
@@ -127,11 +127,11 @@ And as always, feel free to ask questions on the ['Contributors' category on our
 > [!NOTE]
 > The `mobile` directory contains two folders ie. `mobile-api` and `mobile-app`. `mobile-api` contains the API code used for serving the podcasts. `mobile-app` contains the Flutter app which is where you should be when you follow the below steps.
 
-### Configuring dependencies
+### Configuring Dependencies
 
 #### Step 1: Set up the environment variable file
```
Comment (ShaunSHamilton) [inline docs/how-to-setup-freecodecamp-mobile-app-locally.md:132]:
```suggestion
#### Step 1: Set Up the Environment Variable File
```

File: docs/how-to-setup-freecodecamp-locally.md [hunk 16]
```diff
@@ -191,7 +191,7 @@ If you do run into issues, first perform a web search for your issue and see if
 
 And as always, feel free to ask questions on the ['Contributors' category on our forum](https://forum.freecodecamp.org/c/contributors) or [our chat server](https://discord.gg/PRyKn3Vbay).
 
-### Configuring dependencies
+### Configuring Dependencies
 
 #### Step 1: Set up the environment variable file
```
Comment (ShaunSHamilton) [inline docs/how-to-setup-freecodecamp-locally.md:196]:
```suggestion
#### Step 1: Set Up the Environment Variable File
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
I do not care much. That being said, why fix some and not others...

**Comment by Ksound22** [comment]
> 

Help me look out for more. I'll fix them. And even if you're unable to find them, I'll find them, and I'll fix them 😉

File: docs/how-to-setup-freecodecamp-locally.md [hunk 17]
```diff
@@ -118,7 +118,84 @@ This is essential, as it allows you to work on your own copy of freeCodeCamp on
...
+   We highly recommend using [Visual Studio Code](https://code.visualstudio.com/) or [Atom](https://atom.io/). These are great, free, and open source code editors.
+
+4. Set up linting for your code editor.
+
+   You should have [ESLint running in your editor](http://eslint.org/docs/user-guide/integrations.html), and it will highlight anything that doesn't conform to [freeCodeCamp's JavaScript Style Guide](http://forum.freecodecamp.org/t/free-code-camp-javascript-style-guide/19121).
+
+   > [!TIP]
+   > Please do not ignore any linting errors. They are meant to **help** you and to ensure a clean and simple codebase.
+
+## Clone your Fork from GitHub
```
Comment (naomi-lgbt) [inline docs/how-to-setup-freecodecamp-locally.md:198]:
```suggestion
## Clone your Fork from GitHub
```

Did this get duplicated again?

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: docs/how-to-setup-freecodecamp-locally.md (hunk 17 above)
Comment (Ksound22) [reply docs/how-to-setup-freecodecamp-locally.md:198]: Checking now...

Comment (Ksound22) [reply docs/how-to-setup-freecodecamp-locally.md]: Thanks, @naomi-lgbt  for spotting it. I've removed it.

**Review by ShaunSHamilton — APPROVED** [review]
I am happy with this going in as is 👍

**Review by raisedadead — APPROVED** [review]

**Pull request merged**